app/views/shared/_intercom_customer.html.erb

```diff
@@ -0,0 +1,24 @@
+<script type="text/javascript">
+window.intercomSettings = {
+    app_id: "<%=INTERCOM_APP_ID%>",
+    name: "<%= current_user.display_name %>", // Full name
+    email: "<%= current_user.email %>", // Email address
+    created_at: "<%= current_user.created_at.to_i %>", // Signup date as a Unix timestamp
+    owner: true //this is a custom attribute
+  };
+
+<% if controller_name=="dashboard" && action_name=="owner" %>
+	var anchor_value;
+	var stripped_url = document.location.toString().split("#");
+	if (stripped_url.length > 1) {
+	  anchor_value = stripped_url[1]; 
+	  if (anchor_value == "freetrial") {
+	   Intercom('trackEvent', 'trial-signup');
+     }
+    }
```

Comment:
We should consider moving this logic to the ruby, right after the user creation and tagging.